crates/pessimistic-proof/src/proof.rs

```diff
@@ -135,10 +136,31 @@ impl PessimisticProofOutput {
             .with_big_endian()
             .with_fixint_encoding()
     }
+
+    pub fn display_to_hex(&self) -> String {
+        format!(
+            "prev_local_exit_root: {}, prev_pessimistic_root: {}, l1_info_root: {}, \
+             origin_network: {}, consensus_hash: {}, new_local_exit_root: {}, \
+             new_pessimistic_root: {}",
+            Hash(self.prev_local_exit_root),
+            Hash(self.prev_pessimistic_root),
+            Hash(self.l1_info_root),
+            self.origin_network,
+            Hash(self.consensus_hash),
+            Hash(self.new_local_exit_root),
+            Hash(self.new_pessimistic_root),
+        )
+    }
 }
 
 const PESSIMISTIC_CONSENSUS_TYPE: u32 = 0;
 
+const EMPTY_LER: [u8; 32] =
+    hex!("27ae5ba08d7291c96c8cbddcc148bf48a6d68c7974b94356f53754ef6171d757");
+
+const EMPTY_PP_ROOT: [u8; 32] =
+    hex!("2152f3808cb81b33b5a47a7a256d61ab9ea916c66030c405ca9b2aaad3b00f0a");
```

Comment:
Unit tests checking these constants actually correspond to the respective empty roots constructed by constructing the trees and getting the root hash would be nice.